ui/node/node.go

```diff
@@ -17,34 +17,39 @@ import (
 	"github.com/ory/x/stringslice"
 )
 
-// swagger:model uiNodeType
+// swagger:enum Type
 type Type string
 
-// swagger:model uiNodeGroup
-type Group string
-
-func (g Group) String() string {
-	return string(g)
-}
-
 const (
-	DefaultGroup          Group = "default"
-	PasswordGroup         Group = "password"
-	OpenIDConnectGroup    Group = "oidc"
-	ProfileGroup          Group = "profile"
-	RecoveryLinkGroup     Group = "link"
-	VerificationLinkGroup Group = "link"
-	TOTPGroup             Group = "totp"
-	LookupGroup           Group = "lookup_secret"
-	WebAuthnGroup         Group = "webauthn"
-
 	Text   Type = "text"
 	Input  Type = "input"
 	Image  Type = "img"
 	Anchor Type = "a"
 	Script Type = "script"
 )
 
+func (t Type) String() string {
+	return string(t)
+}
+
+// swagger:enum Group
```

Comment:
Could we name this

```suggestion
// swagger:enum UiNodeGroup
```

please?